crates/relayer/src/light_client/eth.rs

```diff
@@ -55,7 +55,7 @@ pub struct ConsensusClient<R: ConsensusRpc> {
     initial_checkpoint: [u8; 32],      // Vec<u8>
     last_checkpoint: Option<[u8; 32]>, // Vec<u8>
     config: Arc<EthChainConfig>,
-    new_block_emitors: Vec<UnboundedSender<Header>>,
+    new_block_emitors: Vec<UnboundedSender<Vec<Header>>>,
```

Comment:
emitter?

Reply:
yes, replace with batch to send multiple headers one time